ONSdigital/eq-author-app
# Added label entry to unit types

### What is the context of this PR?

> https://collaborate2.ons.gov.uk/jira/browse/EAR-830
>
> Remove default type when adding a unit answer

### How to review 

> Add to the list below as appropriate, including screenshots when necessary

* Create questionnaire
* Add answer type of unit
* Check unit type box reads "Select a type"
* Set a unit type
* Check "Select a type" is no longer available in the list

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Jira ticket for this task into the next stage of the process
